doc/source/whatsnew/v0.17.0.txt

```diff
@@ -124,6 +124,36 @@ incrementally.
 
 See the :ref:`docs <io.sas>` for more details.
 
+
+Changes to Excel with ``MultiIndex``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+In version 0.16.2 a ``DataFrame`` with ``MultiIndex`` columns could not be written to Excel via `to_excel`.
+That functionality has been added (:issue:`10564`), along with updating  `read_excel` so that the data can
+be read back with no loss of information by specifying which columns/rows make up the ``MultiIndex``
+in the `header` and `index_col` parameters (:issue:`4679`)
+
+.. ipython:: python
+
+    df = pd.DataFrame([[1,2,3,4], [5,6,7,8]],
+                      columns = pd.MultiIndex.from_product([['foo','bar'],['a','b']],
+                                                           names = ['col1', 'col2']),
+                      index = pd.MultiIndex.from_product([['j'], ['l', 'k']],
+                                                         names = ['i1', 'i2']))
```

Comment:
I like adding your pictures of the before after here as well (the onces from above) (only in the whatsnew)